proxy/http/remap/NextHopStrategyFactory.cc

```diff
@@ -220,9 +236,6 @@ NextHopStrategyFactory::loadConfigFile(const std::string &fileName, std::strings
         std::ifstream file(fileName + "/" + i.data());
         if (file.is_open()) {
           while (std::getline(file, line)) {
-            if (line[0] == '#') {
-              // continue;
```

Comment:
@randall Was this an issue, skipping comment lines?

Reply:
I removed it because the continue wall was commented out. It looked like it was nothing.

Reply:
I reverted that part of the commit just in case you want to add back comment support.